src/components/Lists/AdminList/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import {
+  ContactInfo,
+  IconWrapper,
+  ListItemWrapper,
+  ProfileContacts,
+  StyledEmail,
+  StyledIcons,
+  StyledItemName,
+} from './styles';
+
+import { IconType } from '@/assets/styles/types';
+import { BodySmall, BodyStrong } from '@/assets/styles/typography';
+import { IconSet } from '@/components/Button/Icons';
+import { ProfilePicture, ProfilePicureType } from '@/components/ProfilePicture';
+
+// Фейковые данные
+export const fakeAdmins = [
+  {
+    id: 1,
+    organization_name: 'AirBaltic',
+    email: 'airbaltic@gmail.com',
+  },
+  {
+    id: 2,
+    organization_name: 'British Airways',
+    email: 'britishairways@gmail.com',
+  },
+  {
+    id: 3,
+    organization_name: 'Turkish Airlines',
+    email: 'turkishairlines@gmail.com',
+  },
+  {
+    id: 4,
+    organization_name: 'Emirates',
+    email: 'emirates@gmail.com',
+  },
+  {
+    id: 5,
+    organization_name: 'Cathay Pacific',
+    email: 'cathaypacific@gmail.com',
+  },
+  {
+    id: 6,
+    organization_name: 'Lady Pacific',
+    email: 'ladypacific@gmail.com',
+  },
+  {
+    id: 7,
+    organization_name: 'Bay Pacific',
+    email: 'baypacific@gmail.com',
+  },
+];
```

Comment:
as its reusable components you cant use here use dummy data, dummy data should be on page level and here you need pass it with the props